RockWeb/Blocks/Crm/PersonMerge.ascx.cs

```diff
@@ -410,6 +410,11 @@ protected void lbMerge_Click( object sender, EventArgs e )
                         primaryPerson.SystemNote = GetNewStringValue( "InactiveReasonNote", changes );
                         primaryPerson.SystemNote = GetNewStringValue( "SystemNote", changes );
 
+                        var peopleIds = MergeData.People.Select( a => a.Id ).ToList();
```

Comment:
@shivambareria Instead of having to do another query to get the min created date time. Can you instead just update the MergePerson class to include a CreatedDateTime field and save the value when person is read there (just like it does with ModifiedDateTime). Then you can use a statement here like: 
`primaryPerson.CreatedDateTime = MergeData.People.Min( p => p.CreatedDateTime);`